src/schema/posts.ts

```diff
@@ -1002,6 +1013,31 @@ export const resolvers: IResolvers<any, Context> = {
         },
       );
     },
+    searchQuestionRecommendations: async (
+      source,
+      _,
+      ctx: Context,
+      info,
+    ): Promise<GQLPostQuestion[]> =>
+      graphorm.query(ctx, info, (builder) => ({
+        queryBuilder: builder.queryBuilder
+          .addSelect('random()', 'rand')
```

Comment:
We don't need to add it as a select. We can put it directly in the order by

Reply:
Oh, cool. Nice to know we can do that.